lib/bindings/HTTPBinding.js

```diff
@@ -53,6 +54,7 @@ const { promisify } = require('util');
 const json = promisify(bodyParser.json({ strict: false })); // accept anything JSON.parse accepts.
 const text = promisify(bodyParser.text());
 const raw = promisify(bodyParser.raw());
+const xml = promisify(bodyParser.xml());
```

Comment:
Would you consider removing XML namespace prefixes from tags and attributes? They are not guaranteed to be consistent, they might change from one request to the next. Removing prefixes from tags and attributes would make it easier to parse the message with JEXL later.

This is addressed in https://github.com/Leonidas-from-XIV/node-xml2js/issues/87

```suggestion
const xml2js = require('xml2js');
const xmlStripPrefix = xml2js.processors.stripPrefix;
const xml = promisify(bodyParser.xml({
        xmlParseOptions: {
            // XML namespaces might change from one request to the next.
            // It is useful to remove them from the document,
            // to be able to refer to tags later in JEXL transformations.
            // See https://github.com/Leonidas-from-XIV/node-xml2js/issues/87
            tagNameProcessors: [xmlStripPrefix],
            attrNameProcessors: [xmlStripPrefix]
        }
    })
);
```